Can not install with angular@16.1.0

### Describe the bug

primeng-16.0.0 can not install with angular@16.1.0, in package.json the version is set to ~16.0.0 which exclude any 16.x version:

https://github.com/primefaces/primeng/blob/6fd8a21b3855a2e76282dead9682eabbe84c522f/src/app/components/package.json#L21

this will only allow primeng be installed with angular@16.0.x version.

### Environment

no matter.

### Reproducer

_No response_

### Angular version

16.1.0

### PrimeNG version

16.0.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v18.16.0

### Browser(s)

_No response_

### Steps to reproduce the behavior

_No response_

### Expected behavior

should be installed with angular@16.x

It's not working for me with Angular 16.0.0

Below is npm output and my package.json and package-lock.json

```
PS C:\Users\gotta\Desktop\New folder\Streaming> npm i primeng
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: streaming@0.0.0
npm ERR! Found: @angular/core@16.1.1
npm ERR! node_modules/@angular/core
npm ERR!   peer @angular/core@"16.1.1" from @angular/animations@16.1.1
npm ERR!   node_modules/@angular/animations
npm ERR!     peerOptional @angular/animations@"16.1.1" from @angular/platform-browser@16.1.1
npm ERR!     node_modules/@angular/platform-browser
npm ERR!       peer @angular/platform-browser@"16.1.1" from @angular/forms@16.1.1
npm ERR!       node_modules/@angular/forms
npm ERR!         @angular/forms@"^16.0.0" from the root project
npm ERR!       3 more (@angular/platform-browser-dynamic, @angular/router, the root project)
npm ERR!     @angular/animations@"^16.0.0" from the root project
npm ERR!   peer @angular/core@"16.1.1" from @angular/common@16.1.1
npm ERR!   node_modules/@angular/common
npm ERR!     peer @angular/common@"16.1.1" from @angular/forms@16.1.1
npm ERR!     node_modules/@angular/forms
npm ERR!       @angular/forms@"^16.0.0" from the root project
npm ERR!     peer @angular/common@"16.1.1" from @angular/platform-browser@16.1.1
npm ERR!     node_modules/@angular/platform-browser
npm ERR!       peer @angular/platform-browser@"16.1.1" from @angular/forms@16.1.1
npm ERR!       node_modules/@angular/forms
npm ERR!         @angular/forms@"^16.0.0" from the root project
npm ERR!       3 more (@angular/platform-browser-dynamic, @angular/router, the root project)
npm ERR!     3 more (@angular/platform-browser-dynamic, @angular/router, the root project)
npm ERR!   6 more (@angular/compiler, @angular/forms, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! primeng@"*" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @angular/core@16.0.6
npm ERR! node_modules/@angular/core
npm ERR!   peer @angular/core@"~16.0.0" from primeng@16.0.0
npm ERR!   node_modules/primeng
npm ERR!     primeng@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! C:\Users\gotta\AppData\Local\npm-cache\_logs\2023-06-14T23_27_42_553Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: C:\Users\gotta\AppData\Local\npm-cache\_logs\2023-06-14T23_27_42_553Z-debug-0.log
```
[package.json.txt](https://github.com/primefaces/primeng/files/11751329/package.json.txt)
[package-lock.json.txt](https://github.com/primefaces/primeng/files/11751330/package-lock.json.txt)

--- a/src/app/components/package.json
+++ b/src/app/components/package.json
@@ -18,9 +18,9 @@
     "module": "primeng.js",
     "typings": "primeng.d.ts",
     "peerDependencies": {
-        "@angular/core": "~16.0.0",
-        "@angular/common": "~16.0.0",
-        "@angular/forms": "~16.0.0",
+        "@angular/core": "^16.0.0",
+        "@angular/common": "^16.0.0",
+        "@angular/forms": "^16.0.0",
         "rxjs": "^6.0.0 || ^7.8.1",
         "zone.js": "~0.13.0"
     }

